src/Mailer/Email.php

```diff
@@ -2092,11 +2092,11 @@ protected function _logDelivery($contents)
     
     
     /**
-     * Converts $array to string if array given
-     * @param null $key
+     * Converts array to string
+     * @param array $array
      * @return string
      */
-    protected function flatten($array = null)
+    protected function flatten($array = [])
```

Comment:
What I meant is: it must not have any defaults.

Reply:
It would make sense if it was optional i guess... what a learning process!